css/global.css

```diff
@@ -1013,6 +1020,14 @@ Styled Img Container Components
 
 /* Tablet and smaller 768px */
 @media screen and (max-width: 768px) {
+	/* 
+	===================================
+	Mobile Menu
+	===================================
+	*/
+	header nav .flex-container {
```

Comment:
had some weird issues w mobile menu width. had to explicitly give it width of inherit